backpack/extensions/secondorder/diag_ggn/__init__.py

```diff
@@ -114,6 +114,57 @@ def __init__(self):
         )
 
 
+class BatchDiagGGNExact(BackpropExtension):
```

Comment:
There should be an abstract class `BatchDiagGGN`, which is the parent of the `Exact` and `MC` extensions